engine/team-city-build.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -e
+
+script="$(cd "${0%/*}" 2>/dev/null; echo "$PWD"/"${0##*/}")"
```

Comment:
Maybe true, but it makes the code very shouty and hard to read.  I will change it to uppercase if you have a strong preference